### ISSUE TITLE
Constant DNS lookups

### ISSUE THREAD

**Issue by FDrebin** [signalfx/docker-collectd#45]
When submitting an issue, please provide the following information - Thank you!

- What is the issue you are having?:
    - [Constant DNS lookups to ingest.signalfx.com]
- Which image are you using?:
    - [X] quay.io/signalfuse/collectd (Ubuntu)
    - [] quay.io/signalfuse/collectd-alpine (Alpine)
- How are you getting the image?
    - [X] Pulling and running the hosted images
    - [] Building the image using this github repository
    - [] Extending the hosted image in a custom dockerfile
    If possible, please provide a copy of the dockerfile.
- Are you using the latest version of the image?
    - [X] yes
    - [] no
        - When was the image pulled or built? [2017]
- What version of the docker engine are you using?
    - [Docker version 1.12.1, build 23cf638]
- Copy and paste the command used to run / start the container
  (Please sanitize sensitive information)
 
  ```bash
  >
  ```
- Please describe any customizations made to the image or container
    - What additional plugins have been added?
        - []
    - What additional plugin configurations have been added?
        - []
    - Other (ie added scripts, files, or directories)
        - []
- Please provide any output from the container or container log
```
```

**Comment by codesmith14** [comment]
Hi dxm765,
Could you please elaborate on the issue you are having?  
Specifically:
- What are the run time options that you are passing into the container? _This will help us see what your collectd configurations look like inside the container_
- How you determined that the container is repeatedly conducting DNS look ups?

**Comment by FDrebin** [comment]
Hi charliesignalfx,

Appreciate the swift response, the issue really is that it just seems to be performing these lookups constantly and pointing out to AWS and some other web addresses.

The configuration I used is the default one from the front page:

`docker run --privileged \
  --net="host" \
  -v /etc/hostname:/mnt/hostname:ro \
  -v /proc:/mnt/proc:ro \
  -v /:/hostfs:ro \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -v /etc:/mnt/etc:ro \
  quay.io/signalfuse/collectd`

I determined it was the container after performing WireShark for 20 minutes and looking through both the DNS inquiry logs that coincided with the WireShark log and pointed to my docker host, afterwards I turned off each container 1 by 1 and waited to see when the inquiries stopped; and they only started after the container was started again. 

EDIT: The formatting of the code got messed up, but its the same one on the front page without the API flag.
